### DISCUSSION TITLE
I got the error when yarn install with next 11.1.0

### DISCUSSION THREAD

**Discussion by scw1021** [vercel/next.js#28257] [Help] [answered]
### What version of Next.js are you using?

11.1.0

### What version of Node.js are you using?

14.7.0

### What browser are you using?

Chrome

### What operating system are you using?

Ubuntu

### How are you deploying your application?

Firebase

### Describe the Bug

When I got the following error when I do yarn install with next 11.1.0.
```
Installing using Yarn
yarn install v1.22.4
[1/4] Resolving packages...
[2/4] Fetching packages...
error jest-worker@27.0.0-next.5: The engine "node" is incompatible with this module. Expected version "^10.13.0 || ^12.13.0 || ^14.15.0 || >=15.0.0". Got "14.7.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

### Expected Behavior

I changed next version from 9.0.6  to 11.1.0 in package.json, and created new PR for this change in github.
Then I run cypress in circleci in github, it executes yarn install.
The described error happens.
What is the suitable version of node.js for next 11.1.0?

### To Reproduce

yarn install

**Comment by timneutkens** [comment, marked as answer]
For 12.x later than `12.13.0`. For 14.x later than `14.15.0` based on the versions shown in the message.
